pulpcore/pulpcore/app/settings.py

```diff
@@ -216,64 +265,7 @@
     }
 }
 
-
-def merge_settings(default, override):
-    """
-    Merge override settings into a set of default settings.
-
-    If both default and override have a key that has a dictionary value, these
-    dictionaries are merged recursively. If either of the values are _not_ a
-    dictionary, the override key's value is used.
-    """
-    if not override:
-        return default
-    merged = default.copy()
-
-    for key in override:
-        if key in merged:
-            if isinstance(default[key], dict) and isinstance(override[key], dict):
-                merged[key] = merge_settings(default[key], override[key])
-            else:
-                merged[key] = override[key]
-        else:
-            merged[key] = override[key]
-
-    return merged
-
-
-def load_settings(paths=None):
-    """
-    Load one or more configuration files, merge them with the defaults, and apply them
-    to this module as module attributes.
-
-    Be aware that the order the paths are provided in matters. Settings are repeatedly
-    overridden so settings in the last file in the list win.
-
-    Args:
-        paths: A list of absolute path strings to configuration files in YAML format.
-
-    Returns:
-        dict: The merged settings. This is helpful to see what settings Pulp is contributing, but
-            is not the full set of settings Django uses, as there are a set of Django-provided
-            defaults as well.
-    """
-    settings = _DEFAULT_PULP_SETTINGS
-
-    for path in (paths or []):
-        with suppress(OSError, IOError):
-            # Consider adding logging of some kind, potentially to /var/log/pulp
-            with open(path) as config_file:
-                config = config_file.read()
-                override_settings = yaml.safe_load(config)
-                settings = merge_settings(settings, override_settings)
-
-    for setting_name, setting_value in settings.items():
-        setattr(sys.modules[__name__], setting_name.upper(), setting_value)
-
-    return settings
-
-
-# Read PULP_SETTINGS environment variable to find the location of server.yaml,
-# defaults to /etc/pulp/server.yaml
-PULP_SETTINGS = os.getenv('PULP_SETTINGS', '/etc/pulp/server.yaml')
-load_settings([PULP_SETTINGS])
+# load pulp default settings
+# maybe it is better to define it directly in this settings file?
```

Comment:
I think it would be better to define the settings in this settings file. To load them into another structure and then pull them out again seems unnecessary.